backend/src/main/java/org/example/spring/boot/skeleton/helper/StartUpRunner.java

```diff
@@ -0,0 +1,97 @@
+package org.example.spring.boot.skeleton.helper;
+
+import org.example.spring.boot.skeleton.model.MarketDTO;
+import org.example.spring.boot.skeleton.model.VendorDTO;
+import org.example.spring.boot.skeleton.services.MarketService;
+import org.example.spring.boot.skeleton.services.VendorService;
+import org.example.spring.boot.skeleton.utilities.ProfilePics;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Service;
+
+import java.util.Set;
+
+@Service
+public class StartUpRunner implements CommandLineRunner {
+
+    private final MarketService marketService;
+    private final VendorService vendorService;
+
+    public StartUpRunner(MarketService marketService, VendorService vendorService) {
+        this.marketService = marketService;
+        this.vendorService = vendorService;
+    }
+    @Override
+    public void run(String... args) throws Exception {
```

Comment:
Jövőbeli fejlesztésnek: ezeket a tesztadatokat ki lehetne olvasni egy csv vagy xlsx-ből is